composer.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "tractorcow/silverstripe-fluent",
+    "name": "silverstripe/silverstripe-fluent",
```

That was only added here so our fork could be installed under its own name.

```suggestion
    "name": "tractorcow/silverstripe-fluent",
```